_posts/2024-12-5-sdg_molecularnodes.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "NumFocus Small Development Grant: Advancing Molecular Visualization with MolecularNodes"
+---
+
+<img
+src="{{site.images}}/numfocus.png"
+title="NumFOCUS Foundation" alt="NumFOCUS Foundation"
+style="float: right; width: 10em;" />
+
+We are thrilled to announce that we have been awarded a [Small Development Grant][SDG] by NumFocus to enhance scientific molecular rendering with [MolecularNodes][MN] in 2025. This initiative is a collaborative effort between [Yuxuan Zhuang][Yuxuan] and [Brady Johnston][Brady].
```

Comment:
I'd replace "we" with "MDAnalysis" to make clearer who "we" is (as the grant is given to the MDAnalysis Org — only NF projects are eligible to receive the grant).